CONTRIBUTING.md

```diff
@@ -98,6 +98,7 @@ Similarly, please create one PR per development item, instead of bundling multip
   - If you really need to test the release version (usually not necessary), run `flutter run --release` from the Terminal.
     * The release version cannot be ran in an emulator.
     * You may also need to temporarily change the release signing config. In the [android/app/build.gradle](android/app/build.gradle) file, replace `signingConfig signingConfigs.release` with `signingConfig signingConfigs.debug`.
+    * To switch to debug config on web, in the [web/index.html](web/index.html) file, replace `firebaseConfig.release` with `firebaseConfig.debug`.
```

Comment:
This is kind of "hidden" under a bullet point that's not entirely related.

I think this mention is more related to the table with the exclamation point underneath. I made a suggestion below, though you won't be able to just hit "commit suggestion" since it didn't let me "suggest" on lines 105-106.

Reply:
Oh and maybe we could also add this info in README.md under "building for web".